src/project/local_settings.py

```diff
@@ -54,7 +54,9 @@ def read_env():
 SHAREABOUTS = {
     'FLAVOR': os.environ.get('FLAVOR', 'duwamish_flavor'),
     'DATASET_ROOT': os.environ.get('SITE_URL', 'NO_SITE_URL'),
-    'DATASET_KEY': os.environ.get('SITE_KEY', 'NO_SITE_KEY')
+    'DATASET_KEY': os.environ.get('SITE_KEY', 'NO_SITE_KEY'),
+    'DUWAMISH_DATASET_KEY': os.environ.get('DUWAMISH_DATASET_KEY', 'Error'),
+    'TREES_DATASET_KEY': os.environ.get('TREES_DATASET_KEY', 'Error')
```

Comment:
This is currently hard-coded, of course. We should probably implement this by looping over all of our environment variables that end with `_DATASET_KEY` and `_SITE_KEY`, and loading them into the `SHAREABOUTS` dict.

Assuming that we move our site urls into environment variables, we will add a new variable for each dataset, names `<dataset_id>_SITE_KEY`. Lmk what you think, but I think this is a better way to go.